Library/ViewModels/DiscoveryPostcardViewModelTests.swift

```diff
@@ -483,4 +493,52 @@ internal final class DiscoveryPostcardViewModelTests: TestCase {
     self.fundingProgressBarViewHidden.assertValues([false, false, true, false, false])
     self.fundingProgressContainerViewHidden.assertValues([false, true, false, false, true])
   }
+
+  func testShowsCategoryLabelsExperimental() {
+    let staffPickProject = Project.template
+      |> Project.lens.staffPick .~ true
+      |> Project.lens.category .~ .illustration
+
+    self.vm.inputs.configureWith(project: staffPickProject)
+    self.vm.inputs.configureWith(category: .art)
+    self.vm.inputs.enableProjectCategoryExperiment(true)
+
+    self.projectIsStaffPickViewHidden.assertValue(false)
+    self.projectCategoryStackViewHidden.assertValue(false)
+    self.projectCategoryName.assertValue(KsApi.Category.illustration.name)
+    self.projectCategoryViewHidden.assertValue(false)
+  }
+
+  func testHidesCategoryLabelExperimental() {
+    // Workaround for discrepancy between category ids from graphQL and category ids from the legacy API
+    let categoryId = KsApi.Category.illustration.intID
+    let illustrationCategory = KsApi.Category(id: String(categoryId!), name: KsApi.Category.illustration.name)
```

Comment:
To keep the code style consistent, this line can be replaced for its functional equivalent:
```
let illustrationCategory = KsApi.Category.illustration
                                |> KsApi.Category.lens.id .~ String(categoryId!) 
```